src/index.js

```diff
@@ -1,14 +1,24 @@
 import 'todomvc-common';
 import {TodoModel} from './stores/todoModel';
-import ViewModel from './stores/viewModel';
+import ViewModel, {ALL_TODOS} from './stores/viewModel';
 import TodoApp from './components/todoApp.js';
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {StateNavigator, HTML5HistoryManager} from 'navigation';
 
 var todoModel = new TodoModel('mobx-react-todomvc');
 var viewModel = new ViewModel();
+var stateNavigator = new StateNavigator([
+	{key: 'todo', route: '{filter?}', defaults: {filter: ALL_TODOS}}
+], new HTML5HistoryManager());
 
```

Comment:
Configures the Navigation router with a single route (with the director router, there were three separate routes). Sets a default filter value so that the root Url corresponds to showing 'all' todos. Uses HTML5 history, instead of hash history, to generate user-friendly Urls.